### PR TITLE
connecting tutorials to CMI

### PR DESCRIPTION
**Pull request by antgonza**

### CODE REVIEW

**Comment by antgonza** [comment]
This is ready for review but I think that after reviews (when ready to merge) it will be worth doing a final change: moving the folder tutorials to guides. I didn't do this cause review will be much harder.

cc: @tkosciol

File: qiita_pet/support_files/doc/source/tutorials/account-creation.rst [hunk 1]
```diff
@@ -1,64 +0,0 @@
-.. _account-creation:
```
Comment (tkosciol) [inline qiita_pet/support_files/doc/source/tutorials/account-creation.rst]: CMI workshop doc (http://cmi-workshop.readthedocs.io/en/latest/qiita-create-study.html#signing-up-for-a-qiita-account) is not as comprehensive as this one.
Do you think that we should get rid of retrieving password info, etc.?

File: qiita_pet/support_files/doc/source/tutorials/getting-started.rst [hunk 2]
```diff
@@ -1,328 +0,0 @@
...
----------------------
-
-Studies are the main source of data for Qiita. Studies can contain only one set
-of samples but can contain multiple sets of raw data, each of which can have a
-different preparation. Many experiments will contain only one data set, which
-includes data for all samples. Such experiments will include a single sample
-template and a single prep template.  
-
-However, Qiita can also support more complex study designs. For example
-imagine a study with 100 samples in which:
```
Comment (tkosciol) [inline qiita_pet/support_files/doc/source/tutorials/getting-started.rst]: CMI workshop does not cover those complex designs so far. But there is meta-analysis section (http://cmi-workshop.readthedocs.io/en/latest/qiita-16S-analysis.html#creating-a-meta-analysis).

File: qiita_pet/support_files/doc/source/tutorials/getting-started.rst [hunk 3]
```diff
@@ -1,328 +0,0 @@
...
-   integer.
--  hist\_edge, the edges of each bin in the sequence length histogram
-   stored as a vector of integer.
-
-Once you are happy with these files and you are ready for publication,
-you can contact one of the Qiita admins to submit to EBI, this process normally
-takes a couple of days but can take more depending on availability and how busy
-is the submitting queue.
-
-Study status
```
Comment (tkosciol) [inline qiita_pet/support_files/doc/source/tutorials/getting-started.rst]: Sandbox etc. is not covered by the CMI workshop tutorial.
Should we maybe move it to Qiita philosophy/documentation part? i.e. keep it, but in a different place.

File: qiita_pet/support_files/doc/source/tutorials/account-creation.rst (hunk 1 above)
Comment (antgonza) [inline qiita_pet/support_files/doc/source/tutorials/account-creation.rst]: I think we need to improve the documentation that is in the CMI workshop ... it could be a copy/paste of this.

File: qiita_pet/support_files/doc/source/tutorials/getting-started.rst (hunk 2 above)
Comment (antgonza) [inline qiita_pet/support_files/doc/source/tutorials/getting-started.rst]: Same than before or I can see an argument of leaving this one here? Up to you.

File: qiita_pet/support_files/doc/source/tutorials/getting-started.rst (hunk 3 above)
Comment (antgonza) [inline qiita_pet/support_files/doc/source/tutorials/getting-started.rst]: Sounds reasonable. However, I think Figure 2 of https://qiita.ucsd.edu/static/doc/html/qiita-philosophy/index.html covers it, what do you think?

File: qiita_pet/support_files/doc/source/tutorials/account-creation.rst (hunk 1 above)
Comment (tkosciol) [inline qiita_pet/support_files/doc/source/tutorials/account-creation.rst]: Agreed! Working on it on https://github.com/biocore/cmi-workshops side. Will ask you for review.

File: qiita_pet/support_files/doc/source/tutorials/getting-started.rst (hunk 2 above)
Comment (tkosciol) [inline qiita_pet/support_files/doc/source/tutorials/getting-started.rst]: The reason we don't have it in CMI workshop tutorial is that we don't use it with the test data that we provide, i.e. the idea is to show each step with example data.
I'd be happy to move it to CMI tutorial if/when we have a more complex dataset for people to work on.
In the current format, I'd rather not complicate the dataset, but we were discussing "intermediate" or "advanced" tutorial and that would fit perfectly there.

tl;dr: let's keep it in Qiita documentation for now.

Comment (antgonza) [inline qiita_pet/support_files/doc/source/tutorials/getting-started.rst]: OK, will add it to the philosophy page ...

Comment (antgonza) [inline qiita_pet/support_files/doc/source/tutorials/getting-started.rst]: @tkosciol ... I just added it, had to make some changes so it fits better in that page

**Comment by antgonza** [comment]
cc-ing: @adswafford

**Comment by antgonza** [comment]
ping @tkosciol and @adswafford

**Comment by tkosciol** [comment]
Sorry! Will review before the end of the day.

**Comment by adswafford** [comment]
Don't know the etiquette, and can't find any buttons to click to approve, but the changes look good to me. Maybe I can get a quick tutorial on this tomorrow?

**Comment by antgonza** [comment]
@tkosciol, you should have permissions now ... thanks!

**Comment by antgonza** [comment]
forgot to say: @adswafford, let me know if you want to chat, just send me an email ...
